datumaro/components/exporter.py

```diff
@@ -161,6 +162,11 @@ def patch(cls, dataset, patch, save_dir, **options):
 
         return retval
 
+    def _apply(self):
+        if self._save_hashkey_meta:
+            self._save_hashkey_file(self._save_dir)
+        self.apply()
```

Comment:
This can be conflicted with
https://github.com/openvinotoolkit/datumaro/blob/565b95d36df9bbd681247b7e24c64fe5acaf09e1/datumaro/plugins/data_formats/arrow/exporter.py#L312

In addition, it seems unnatural for a private function (`_apply()`) to call a public function (`apply()`) with the same name.